choreolib/src/main/native/include/choreo/lib/ChoreoTrajectoryState.h

```diff
@@ -91,6 +99,12 @@ class ChoreoTrajectoryState {
   /// The angular component of the velocity at that point in the trajectory
   units::radians_per_second_t angularVelocity = 0_rad_per_s;
 
+  /// The forces on the modules in the X direction
+  std::array<units::newton_t, 4> moduleForcesX{{0_N, 0_N, 0_N, 0_N}};
+
+  /// The forces on the modules in the Y direction
+  std::array<units::newton_t, 4> moduleForcesY{{0_N, 0_N, 0_N, 0_N}};
```

Comment:
Do we wanna use `wpi::SmallSet` or something similar here?

Reply:
Why?